draft-ietf-tls-dtls-connection-id.md

```diff
@@ -51,22 +51,25 @@ author:
 
 normative:
   RFC2119:
-  I-D.ietf-tls-dtls13:
-  RFC6347:
   RFC5246:
+  RFC6347:
 informative:
   RFC6973:
+  I-D.ietf-tls-dtls13:
+
 --- abstract
 
-This document specifies the "Connection ID" concept for the Datagram Transport Layer Security
-(DTLS) protocol, version 1.2 and version 1.3.
+This document specifies the Connection ID construct for the Datagram Transport
+Layer Security (DTLS) protocol version 1.2.  {{I-D.ietf-tls-dtls13}} specifies
+the Connection ID for DTLS version 1.3.
 
-A Connection ID is an identifier carried in the record layer header that gives the
-recipient additional information for selecting the appropriate security association.
-In "classical" DTLS, selecting a security association of an incoming DTLS record
-is accomplished with the help of the 5-tuple. If the source IP address and/or
-source port changes during the lifetime of an ongoing DTLS session then the
-receiver will be unable to locate the correct security context.
+A Connection ID is an identifier carried in the record layer header that gives
+the recipient additional information for selecting the appropriate security
+association.  In "classical" DTLS, selecting a security association of an
+incoming DTLS record is accomplished with the help of the 5-tuple.  If the
+source IP address and/or source port changes during the lifetime of a DTLS
+session then the receiver will be unable to locate the correct security
+context.
```

Comment:
Is this jsut reflow?

Reply:
Yep, this must be me inadvertently doing a `gq` in vim.  I'll revert it.